Gemfile

```diff
@@ -92,3 +92,7 @@ gem 'cancancan', '~> 1.9'
 
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
+
```

Comment:
once again this can be put together with the gems at the top, after migration comments, I think.